### PR TITLE
Fix teamsMediaLoggingPolicy related links

### PR DESCRIPTION
**Pull request by LeoKuhorev**
Fix related links for teamsMediaLoggingPolicy cmdlets

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md [hunk 1]
```diff
@@ -127,4 +127,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## RELATED LINKS
 
-[https://docs.microsoft.com/en-us/powershell/module/skype/get-csteamsmedialoggingpolicy](https://docs.microsoft.com/en-us/powershell/module/skype/get-csteamsmedialoggingpolicy)
+[Grant-CsTeamsMediaLoggingPolicy](https://docs.microsoft.com/en-us/powershell/module/teams/grant-csteamsmedialoggingpolicy)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsMediaLoggingPolicy.md:130]:
```suggestion
[Grant-CsTeamsMediaLoggingPolicy](/powershell/module/teams/grant-csteamsmedialoggingpolicy)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @LeoKuhorev, kindly commit the suggested changes to change the links to relative links. Many thanks.

**Review by get-itips — APPROVED** [review]
I am ok after @JohanFreelancer9 suggestions applied

**Review by dariomws — APPROVED** [review]

File: teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md [hunk 2]
```diff
@@ -229,4 +229,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## RELATED LINKS
 
-[https://docs.microsoft.com/en-us/powershell/module/skype/grant-csteamsmedialoggingpolicy](https://docs.microsoft.com/en-us/powershell/module/skype/grant-csteamsmedialoggingpolicy)
+[Get-CsTeamsMediaLoggingPolicy](https://docs.microsoft.com/en-us/powershell/module/teams/get-csteamsmedialoggingpolicy)
```
[the remark this answers is not in the document]
Comment (LeoKuhorev) [reply teams/teams-ps/teams/Grant-CsTeamsMediaLoggingPolicy.md]: Thanks for the suggestion, applied!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**